centos-ci/libtask.sh

```diff
@@ -18,9 +18,6 @@ prepare_job() {
 
     . ~/rsync-password.sh
 
-    # Ensure we're operating on a clean base
-    (cd ${buildscriptsdir} && git clean -dfx && git reset --hard HEAD)
```

Comment:
Hmm, I'd prefer `rsync --delete` over doing a dummy commit. Any reason we don't do this?